modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectLayoutLocalServiceTest.java

```diff
@@ -449,7 +449,7 @@ public void testGetObjectLayout() throws Exception {
 	@Test
 	public void testUpdateObjectLayout() throws Exception {
 		List<ScreenNavigationCategory> screenNavigationCategories =
-			_screenNavigationRegistry.getScreenNavigationCategories(
+			ScreenNavigationRegistryUtil.getScreenNavigationCategories(
```

Comment:
Hey @GabrielHenriquedev, I don't think this works. Please try to use Reflection to fix this test, thanks.